lib/win-rebase.js

```diff
@@ -1,8 +1,17 @@
 var re = /^\s*("*)([^"]*?\b(?:node|iojs)(?:\.exe)?)("*)( |$)/
+var npmre = /^\s*("*)([^"]*?\b(?:npm))("*)( |$)/
+var path_ = require('path')
+if (path_.win32) path_ = path_.win32
 
-module.exports = function (path, rebase) {
+module.exports = function (path, rebase, whichOrUndefined) {
```

Comment:
Yeah, that thought crossed my mind, and tbh the reason to have it this way was that it made mocking it for the test file a bit easier…